pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -0,0 +1,151 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, fetchurl
+, runCommand
+, patchutils
+, pkg-config
+, glibcLocales
+, cmake
+, python3
+, libpng
+, zlib
+, eigen
+, protobuf
+, nsync
+, howard-hinnant-date
+, re2
+, nlohmann_json
+, boost
+, oneDNN
+}:
+
+let
+  externals = {
```

Comment:
Could `fetchSubmodules` be used instead? Manually having to update this seems like a bit of a pain.

Reply:
switch to fetchSubmodules